notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1249 @@
+{
```

Comment:
<p> </p><p>I think this part is more related to general openvino API and can be removed:</p><p><s>Any of the device names returned by the </s><code><s>available_devices</s></code><s> method are valid device specifiers.   </s></p><p><s>You may also use "AUTO", which will automatically select the best device for inference.     To learn more about AUTO plugin, visit the </s><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/auto-device-selection.html" target="_blank"><s>Automatic Device Selection</s></a><s> page.</s></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>